app/services/manifest_validator.rb

```diff
@@ -0,0 +1,28 @@
+class ManifestValidator
+  include JsonSchemaHelper
+
+  attr_reader :content, :errors
```

Comment:
do we need this `attr_reader` to be public? looks like we are only using these ivars internally